e2e/cypress/integration/messaging/private_channel_open_spec.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// **************************    ***********************************
```

Comment:
nit: please connect :)

Reply:
Added a commit that fixes the formatting